app/views/login.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="OWASP NodeGoat Project: Insecure App">
+    <meta name="version" content="v1.2">
+
+    <title>OWASP Node Goat</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="vendor/bootstrap/bootstrap.css" rel="stylesheet">
+
+    <!-- Theme CSS -->
+    <link href="vendor/theme/sb-admin.css" rel="stylesheet">
+    <link rel="stylesheet" href="vendor/theme/font-awesome/css/font-awesome.min.css">
+
+    <style type="text/css">
+    #wrapper {
+        padding-left: 0;
+    }
+    </style>
+
+</head>
+
+<body>
+
+    <div id="wrapper">
+        <!-- Sidebar -->
+        <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <a class="navbar-brand" href="https://www.owasp.org/index.php/Projects/OWASP_Node_js_Goat_Project">OWASP Node Goat Project</a>
+            </div>
+            <!--  Nav Links-->
+            <div class="collapse navbar-collapse navbar-ex1-collapse">
+                <!-- top nav -->
+                <ul class="nav navbar-nav navbar-right navbar-user">
+
+                    <li class="dropdown user-dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-size: larger"><i class="fa fa-info-circle"></i></a>
+                        <ul class="dropdown-menu alert-dropdown" style="min-width: 350px; padding: 10px">
+                            <li>
+                                <p>
+                                    The OWASP Node Goat is an educational Node.js web application vulnerable to the <a target="_blank" href="https://www.owasp.org/index.php/Top_10_2013-Top_10"> OWASP Top 10</a> risks.</p>
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 77880d7ceb2be5c290a80a754f7c8357 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Insecure Link Target
_Low CWE-1022_
The application uses an anchor tag whose `target` attribute is set to `_blank`, but the `rel` attribute is not set with the `noopener` value. The page opened from that target link may have the access to the `location` object of the original page through `window.opener.location`. A malicious page associated with this link will be able to navigate the original page to an arbitrary site, such as a phishing site, and thus, execute a reverse tabnabbing attack.
Note that most contemporary browsers add the `noopener` value to the `rel` automatically by default. Thus, only users with older versions of browsers are targets for this phishing attack.



### How to fix
To prevent untrusted pages linked from the application from being able to manipulate the "window.opener" object, set the attribute rel="noopener noreferrer" for each HTML link to external sites. In addition to preventing reverse tabnabbing attacks, it may also improve browser performance as the new page can be run in a separate process.

When the rel attribute is set to "noopener", the "window.opener" object is "null" on the new page. Therefore, the attacker cannot manipulate the navigation of the original page.  Additionally, the "noreferrer" attribute tells the browser to not include referrer information when the new page is requested.

For links opened with JavaScript, add the values "noopener", "noreferrer" as the "windowFeatures" parameter in the "window.open()" function:
```
var childWindow = window.open(url, name, "noopener,noreferrer");   
```
For legacy browser support, you can also manually set the opener to "null":
```
childWindow.opener = null;  
```

**Note**: The rel="noopener" attribute setting is supported by most modern browsers such as Firefox, Chrome, Edge and Safari but is not supported by legacy browsers such as Internet Explorer. For the latest information on browser support, refer to https://caniuse.com/#search=rel.